### ISSUE TITLE
Llama3.1 Latex format is not showing correctly

### ISSUE THREAD

**Issue by JianhaoLuo18** [chathub-dev/chathub#1251]
**Describe the bug**
![Uploading Screenshot_2024-07-31_09-24-27.png…]()

**Desktop (please complete the following information):**
 - OS: [e.g. Windows]
 - Browser [e.g. chrome, brave]
 - ChatHub Version [you can find it in ChatHub Settings]

**Comment by wong2** [comment]
The screenshot is broken

**Comment by JianhaoLuo18** [comment]
![Uploading Screenshot_2024-07-31_10-47-48.png…]()
let me try again

**Comment by JianhaoLuo18** [comment]
![Screenshot_2024-07-31_10-47-48](https://github.com/user-attachments/assets/89a48430-7d3f-4f7f-a17a-4384c24cb1be)
ok check above, z score formula, obviously is not rendered.

**Comment by wong2** [comment]
Does only llama 3.1 encounter this issue? Are the other models working fine?

**Comment by JianhaoLuo18** [comment]
Chatgpt 4o is working correctly 
Claude not either
